src/libcore/intrinsics.rs

```diff
@@ -1173,15 +1563,24 @@ extern "rust-intrinsic" {
     pub fn frem_fast<T>(a: T, b: T) -> T;
 
     /// Convert with LLVM’s fptoui/fptosi, which may return undef for values out of range
-    /// https://github.com/rust-lang/rust/issues/10184
+    /// (<https://github.com/rust-lang/rust/issues/10184>)
```

Comment:
Not sure if the link above is strictly necessary, since intrinsics are already unsafe, but if we want to keep it then let's add a little context to the link so that people are more likely to click it, e.g. `Extra care must be taken to avoid undefined behavior: <https://github.com/rust-lang/rust/issues/10184>`.

Reply:
I didn't want to change this, because it was already there (I only made the link active), but I will consider looking for a better wording

Reply:
we can do this separately, don't want to block this PR on this. thanks